### DISCUSSION TITLE
Soy nuevo utilizando Github

### DISCUSSION THREAD

**Discussion by pandafrontdeveloper** [community/community#81214] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

¿Podrían dejarme algunas recomendaciones para poder entender las ramas?

**Comment by ignaciodiazdev** [comment, marked as answer]
¡Crea ramas con git branch, cambia con git checkout. Fusiona cambios con git merge. Nombra claro. Practica en proyectos pequeños.

**Comment by ananuness** [comment]
Seleccioné dos videos interesantes que hablan sobre las ramas y también la [documentación de Github](https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) en sí, este puede ser más denso, ¡pero aún así puede ayudar mucho! Espero que te ayude y cualquier cosa ¡házmelo saber!

- [¿Que son las ramas?](https://youtu.be/LL47l59Ld5I?si=50aSahV656Jzlws1)
- [Git: todo lo que tenés que saber sobre branches (ramas)](https://youtu.be/gjKKtQVVCZU?si=J8feqvwEz3JUJAC6)
